test/index.js

```diff
@@ -11,10 +11,11 @@ db.addMiddleware(addTimestamps);
 describe('Middleware timestamps', () => {
 	let dummyCollection;
 	before(() => {
+		db.create('dummy');
 		dummyCollection = db.get('dummy');
 	});
 	after(() => {
```

Comment:
I am not sure if my comment about async handling applies here as well. Chances are it is.